startup_test.go

```diff
@@ -25,6 +27,34 @@ func TestIsBotRunning(t *testing.T) {
 	}
 }
 
+func TestDiscord(t *testing.T) {
+	botId, err := getBotID()
+	if err != nil {
+		t.Fatalf("could not get botId: %v", err)
+	}
+	pw, err := ioutil.ReadFile(".password")
+	if err != nil {
+		t.Fatalf("could not read password file")
+	}
+	apitoken := getDiscordToken()
+	if apitoken == "" {
+		t.Fatalf("could not read discord token env")
+	}
+	token, err := login("admin", string(pw), *botId)
+	if err != nil {
+		t.Fatalf("could not get token: %v", err)
+	}
+	instance, err := createDiscordInstance(string(apitoken), *token)
+	if err != nil {
+		t.Fatalf("could not create instance: %v", err)
+	}
+	fmt.Println("Created instance ", instance.UUID)
+
+	if err := spawnInstance(instance.UUID, *token); err != nil {
+		t.Fatalf("could not spawn discord instance: %v", err)
+	}
```

Comment:
we should assert here that the bot actually joined discord. So an API call to discord would make sense.

Reply:
Will do